src/components/PlaidLink/index.native.tsx

```diff
@@ -1,27 +1,28 @@
 import {useEffect} from 'react';
-import {dismissLink, openLink, useDeepLinkRedirector, usePlaidEmitter} from 'react-native-plaid-link-sdk';
+import {dismissLink, LinkEvent, openLink, useDeepLinkRedirector, usePlaidEmitter} from 'react-native-plaid-link-sdk';
 import Log from '@libs/Log';
 import CONST from '@src/CONST';
-import {plaidLinkDefaultProps, plaidLinkPropTypes} from './plaidLinkPropTypes';
+import PlaidLinkProps from './types';
 
-function PlaidLink(props) {
+function PlaidLink({token, onSuccess = () => {}, onExit = () => {}, onEvent}: PlaidLinkProps) {
     useDeepLinkRedirector();
-    usePlaidEmitter((event) => {
-        Log.info('[PlaidLink] Handled Plaid Event: ', false, event);
-        props.onEvent(event.eventName, event.metadata);
+    usePlaidEmitter((event: LinkEvent) => {
+        Log.info('[PlaidLink] Handled Plaid Event: ', false, event.eventName);
+        onEvent?.(event.eventName, event.metadata);
```

Comment:
@JKobrynski I've found only one usage of [this component](https://github.com/Expensify/App/blob/main/src/components/AddPlaidBankAccount.js#L205), and it has `onEvent` prop, so maybe we can make it required and get rid of chaining.

Reply:
Great idea, done!